### PR TITLE
services: Pull images from quay.io instead of docker.io

### LINKED ISSUE #1418: Image refresh for services
Image refresh for services
 * [ ] FAIL: image-refresh services

### PR DESCRIPTION
**Pull request by mvollmer**
We keep up-date mirrors of them there, and avoid running into
dockerhub.io limits during image creation.

 * [x] image-refresh services

Fixes #1418

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 2-cockpit-10.
Log: https://logs.cockpit-project.org/logs/image-refresh-1422-20201130-133636/

File: sync-quay [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#! /bin/sh
+
+set -x
```
Comment (martinpitt) [inline sync-quay:3]: `set -eu` as well please

File: .github/workflows/daily.yml [hunk 2]
```diff
@@ -16,6 +16,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Log into quay
+        run: docker login -u="${{ QUAY_BOTUSER }}" -p="${{ secrets.QUAY_TOKEN }}" quay.io
+
+      - name: Synchronize container image mirror
+        run: ./sync-quay
```
Comment (martinpitt) [inline .github/workflows/daily.yml:23]: I wouldn't mind if that was inline for simplicity. You can do the normal YAML thing:
```yaml
    run: |
        shell
        .. script
```
```

**Comment by cockpituous** [comment]
image-refresh services done: https://github.com/cockpituous/bots/commits/image-refresh-services-20201130-142020

**Comment by mvollmer** [comment]
Succesfull action run: https://github.com/cockpit-project/bots/runs/1474394810

**Comment by mvollmer** [comment]
"services" image has been created successfully.  Do we want to include it here?

**Comment by martinpitt** [comment]
>  Do we want to include it here?

I don't see why not -- it's a bona fide refresh, no? And it'd be good to self-validate.

**Review by martinpitt — DISMISSED** [review]
Looks great now, thanks! Just needs a Closes# (or we stop bothering, of course -- fine for me!)

**Comment by mvollmer** [comment]
> Looks great now, thanks! Just needs a Closes# (or we stop bothering, of course -- fine for me!)

I added it, with push-rewrite... could you approve again?

File: .github/workflows/daily.yml (hunk 2 above)
Comment (martinpitt) [reply .github/workflows/daily.yml:23]: You mean you want to run it locally as well? Then let's keep it as a script for now indeed. Bikeshedding :-)

**Review by martinpitt — APPROVED** [review]

**Comment by martinpitt** [comment]
@mvollmer: Ah, this doesn't have a "Fixes #1418", so you need to close that issue manually after landing that.

**Comment by martinpitt** [comment]
@mvollmer: I added the Fixes # to the PR description. AFAIR this should work, let's find out when you press the button.

**Pull request merged**